Idle_Game/app/src/main/java/com/example/idle_game/data/repositories/GameRepository.kt

```diff
@@ -67,12 +74,21 @@ class GameRepository(
                     accessToken = null
                 )
                 gameDao.insertPlayer(playerData)
+                try {
+                    inventoryDataFlow.first().bitcoins //Try to get Data out of existing? db
+                } catch (e: Exception) {
+                    createNewInventory()
+                }
             }
         } catch (e: HttpException) {
             onFailure()
         }
     }
 
+    suspend fun DEBUG_ABMELDEN() {
```

Comment:
Was ist das für eine Methode, warum ist random alles auf deutsch?

Reply:
Oh die hab ich übersehen. War eigentlich nur für micht gedacht

Reply:
Ich lasse die Methode doch drin, schonmal für die settings page
ich habe sie aber in logout umbenannt.